validate.js

```diff
@@ -0,0 +1,14 @@
+const submitButton = document.getElementById('submit');
+const email = document.getElementById('mail');
+
+email.value = 'FErnando@gmail.com';
```

Comment:
I think you do not need to give a default email address. You should only validate based on whatever input available on the `email` field at any point in time when the submit button is clicked.